src/tokenizer.rs

```diff
@@ -562,7 +566,16 @@ impl<'a> Tokenizer<'a> {
                         s += s2.as_str();
                         return Ok(Some(Token::Number(s, false)));
                     }
-                    Ok(Some(Token::make_word(&s, None)))
+
+                    // https://dev.mysql.com/doc/refman/8.0/en/charset-introducer.html
+                    if ch == '_' && dialect_of!(self is MySqlDialect) {
+                        const INTRODUCERS: [&str; 4] = ["_latin1", "_binary", "_utf8", "_utf8mb4"];
```

Comment:
My reading of https://dev.mysql.com/doc/refman/8.0/en/charset-introducer.html suggests the value for the charset introducer can be any character set supported; Why are these 4 hard coded?

Reply:
That's good point! I'll fix it.